### PR TITLE
Update Pegasus Importing Data docs

### PR DESCRIPTION
**Pull request by kelbyhawn**
Updates instructions on `importing-data.md` to import data in Pegasus from a `cdo-` gsheet.

**Related PR:** https://github.com/code-dot-org/code-dot-org/pull/55976

**Jira ticket:** [ACQ-1436](https://codedotorg.atlassian.net/browse/ACQ-1436)

### CODE REVIEW

File: docs/importing-data.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
-Data may be imported from Google Sheets or CSV files into SQL tables. Columns in these spreedsheets must be named using type suffixes.
+Data may be imported from Google Sheets into SQL tables. Columns in these spreedsheets must be named using type suffixes.
 
 ### Importing Google Sheets
 
-1. Create a sheet in the "Pegasus" Google Docs folder, naming your columns using type suffixes.
-1. In the Dropbox folder for a site, create a file named `data/table_name.gsheet` where table_name is the name you'd like the database table to have. The file should contain the "Pegasus" Google Docs path to the sheet you created in step 1. For example, `code.org/data/about_team.gsheet` would contain `Data/About/Team` and the data will be imported into a table named `about_team`.
-1. The data should import on staging automatically within a few minutes.
-
-### Importing CSV files
```
Comment (kelbyhawn) [inline docs/importing-data.md:13]: This file was last updated 10 years ago, and I don't think importing CSV files is relevant anymore.

Comment (kelbyhawn) [reply docs/importing-data.md:13]: @bethanyaconnor is this something you might know about? No worries if not!

Comment (bethanyaconnor) [reply docs/importing-data.md:13]: Not 100% sure but I believe we only import CSV files from google sheets now. I think we looked and the `data/` folder doesn't exist right? So either way, this documentation isn't useful, so I'd remove it. We can add documentation if we do discover this

File: docs/importing-data.md [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
-
-1. In the Dropbox folder for a site, create a file named `data/table_name.csv` where table_name is the name you'd like the database table to have naming your columns using type suffixes.
+1. Create a new gsheet in the "Pegasus" Google Docs folder, naming your columns using type suffixes.
+  - Make SiteCloud (`site.googlecloud@code.org`) the owner of the gsheet.
```
Comment (bethanyaconnor) [inline docs/importing-data.md:10]: Thank you for adding this!

**Review by bethanyaconnor — APPROVED** [review]
Thank you for updating this!

File: docs/importing-data.md (hunk 1 above)
Comment (kelbyhawn) [reply docs/importing-data.md:13]: Ok cool, ty!

Comment (TurnerRiley) [reply docs/importing-data.md:13]: I think we also import [CSV files from AWS](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/census/census_summary.rb#L43) too! (Not that this changes if this documentation should be deleted)

**Pull request merged**